### PR TITLE
Update rubygem-foreman_puppet to 4.0.3

### PR DESCRIPTION
**Pull request by Ron-Lavi**

### CODE REVIEW

File: packages/plugins/rubygem-foreman_puppet/rubygem-foreman_puppet.spec [hunk 1]
```diff
@@ -9,7 +9,7 @@
 %global release 2
```
Comment (ehelms) [inline packages/plugins/rubygem-foreman_puppet/rubygem-foreman_puppet.spec:9]: Needs to be set back to 1

Comment (Ron-Lavi) [reply packages/plugins/rubygem-foreman_puppet/rubygem-foreman_puppet.spec]: Updated 👍🏼 btw what does `release 1/2` mean?

Comment (ekohl) [reply packages/plugins/rubygem-foreman_puppet/rubygem-foreman_puppet.spec]: Don't forget the changelog entry. That must match.

The release is a version number for the packaging. It allows packagers to make incremental changes on the same original release. https://github.com/theforeman/foreman-packaging/commit/3e4c0f1924a8760df55b68a0fe826f48f34c3b32 is an example of that.

Comment (Ron-Lavi) [reply packages/plugins/rubygem-foreman_puppet/rubygem-foreman_puppet.spec]: Thanks for the explanation, done.

**Comment by LiorKGOW** [comment]
Correct me if I am wrong :angel:, I think the needs testing label could be removed

**Review by LiorKGOW — APPROVED** [review]

**Review by ekohl — APPROVED** [review]

**Comment by ekohl** [comment]
> Correct me if I am wrong angel, I think the needs testing label could be removed

I did ask about that in the past (https://community.theforeman.org/t/drop-needs-testing-label-in-prs/22504) but never moved forward with it.

**Pull request merged**